### PR TITLE
Add basics for inventory system

### PR DESCRIPTION
**Pull request by kthompson**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+# See https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/code-style-rule-options?view=vs-2019  for supported options
+[*.cs]
+indent_size = 4
+indent_style = tab
+
+dotnet_analyzer_diagnostic.category-Globalization.severity = none
+
+
+# Code Analysis tweaks for style rules
+dotnet_diagnostic.IDE0008.severity = none
```
Comment (kthompson) [inline .editorconfig:17]: i just copied all of these from a work project so if the styles are annoying we can nuke them.

File: Assets/Inventory/GameData.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (kthompson) [inline Assets/Inventory/GameData.json:1]: game data is all possible items in the game

File: Assets/Inventory/PlayerData.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[
```
Comment (kthompson) [inline Assets/Inventory/PlayerData.json:1]: player data is what the user has in the inventory. we may want to save this in the user profile or just load some basic defaults. there is no persistence atm

**Review by parkernalch — APPROVED** [review]
looking forward to trying this out

**Comment by kthompson** [comment]
> looking forward to trying this out

Cool. its still super basic next steps are adding a toolbar and doing some kind drag and drop

**Pull request merged**